app/assets/stylesheets/modules/edit_form.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+@import "mixins";
+
+.form {
+
+  .legend{
+    font-family:$dogstay-font-theme;
+    font-size: 2em;
+  }
+
+  img {
+    @include image-style;
+    margin-top: 3em;
+    margin-left: 0;
+    margin-right: 0;
+    height: 15em;
+    width: 15em;
+  }
+
+  ul {
+    font-family: $sitter-font-type;
+    padding-left:2.2em;
+  }
+
+  .postfix {
+    width:30%;
+    margin-left: auto;
+    margin-right: auto;
+    background: teal;
```

Comment:
Color `teal` should be written in hexadecimal form as `#008080`<br>Color literals like `teal` should only be used in variable declarations; they should be referred to via variable everywhere else.